packages/pn-pa-webapp/src/pages/TermsOfService.page.tsx

```diff
@@ -1,53 +1,168 @@
-import { Fragment, ReactNode, useEffect } from 'react';
-import { useNavigate } from 'react-router-dom';
-import { Trans } from 'react-i18next';
-import { Grid } from '@mui/material';
-import { TermsOfServiceHandler, useIsMobile } from '@pagopa-pn/pn-commons';
+import {
+  Box, Container,
+  Link,
+  Typography
+} from "@mui/material";
 
-import { useAppDispatch, useAppSelector } from '../redux/hooks';
-import { acceptToS } from '../redux/auth/actions';
-import * as routes from '../navigation/routes.const';
-import { RootState } from '../redux/store';
+const PrivacyPolicyPage = () => (
```

Comment:
Check if the page must be localized

Reply:
The terms of service and privacy policy pages don't need to be localized, as they are placeholder text and will be injected through a script in the future and managed through a cms by the legal team, as far as I know.